libs/search-widget/src/widget.svelte

```diff
@@ -179,23 +179,23 @@
 <style>
   @import url('https://fonts.googleapis.com/css2?family=Source+Sans+Pro:wght@300;400;700&display=swap');
   .nuclia-widget {
-    --color-primary-regular: var(--custom-color-primary-regular, black);
-    --color-primary-muted: var(--custom-color-primary-muted, #707070);
+    --color-primary-regular: var(--custom-color-primary-regular, hsl(336, 100%, 50%));
+    --color-neutral-strong: var(--custom-color-neutral-strong, hsl(0, 0%, 44%));
+    --color-neutral-regular: var(--custom-color-neutral-regular, hsl(0, 0%, 77%));
+    --color-neutral-lightest: var(--custom-color-neutral-lightest, hsl(240, 7%, 97%));
     --color-light-stronger: var(--custom-color-light-stronger, white);
-    --color-dark-light: var(--custom-color-dark-stronger, #f7f7f8);
-    --color-dark-stronger: var(--custom-color-dark-stronger, #8296a6);
-    --color-neutral-strong: var(--custom-color-neutral-strong, #ff0066);
-    --color-backdrop: var(--custom-color-backdrop, rgba(0, 0, 0, 0.5));
-    --color-light-regular: var(--custom-color-light-regular, #c4c4c4);
+    --color-dark-stronger: var(--custom-color-dark-stronger, hsl(0, 0%, 5%));
     --color-secondary-stronger: var(--custom-color-secondary-stronger, hsl(51, 78%, 32%));
     --color-secondary-lightest: var(--custom-color-secondary-lightest, hsl(51, 100%, 95%));
 
+    --color-blue-strong: hsl(207, 17%, 58%);
```

Comment:
Why didn't you use on of the blue colors we have in Sistema palette?
```scss
// Tertiary
$color-tertiary-stronger:   hsl(249, 100%, 24%);
$color-tertiary-strong:     hsl(249, 100%, 40%);
$color-tertiary-regular:    hsl(249, 100%, 50%); // #2600FF
$color-tertiary-light:      hsl(249, 100%, 65%);
$color-tertiary-lighter:    hsl(249, 100%, 92%);
$color-tertiary-lightest:   hsl(249, 100%, 96%);
```

Reply:
Fixed